homeassistant/components/samsungtv/config_flow.py

```diff
@@ -24,21 +21,20 @@
 )
 
 # pylint:disable=unused-import
-from .const import CONF_MANUFACTURER, CONF_MODEL, DOMAIN, LOGGER
+from .bridge import SamsungTVLegacyBridge, SamsungTVWSBridge
```

Comment:
Why do we have a base class, when we need the actual implementations?

Reply:
They will disappear following your above suggestion